.github/workflows/on-demand-build-docker-image-deploy-preview.yml

```
@@ -5,52 +5,79 @@ on:
+            PR: ${{ needs.resolve-params.outputs.pr_number }}.
             recreate: ${{ github.event.client_payload.slash_command.args.named.recreate }}.
 
   server-build:
+    needs: [resolve-params]
     name: server-build
     if: github.event.client_payload.slash_command.args.named.env != 'release'
     uses: ./.github/workflows/server-build.yml
     secrets: inherit
     with:
-      pr: ${{ github.event.client_payload.pull_request.number }}
+      pr: ${{ needs.resolve-params.outputs.pr_number }}
```

actionlint (1.7.11)

[error] input "pr" is typed as number by reusable workflow "./.github/workflows/server-build.yml". string value cannot be assigned

(expression)